decidim-dev/lib/decidim/dev/test/spec_helper.rb

```diff
@@ -32,7 +32,7 @@
   # instead of true.
   config.use_transactional_fixtures = true
 
-  config.include Rectify::RSpec::Helpers
+  # config.include Rectify::RSpec::Helpers
```

Comment:
Please remove the commented line if it is not needed anymore.

I believe the RSpec helpers are not needed (as the tests are passing with this commented out)?